core/json/locales/EN/translations.json

```diff
@@ -33,21 +38,29 @@
 "GYMS_METATITLE": "Follow the conquest",
 "GYMS_OWNED": "gyms owned",
 "GYMS_TITLE": "The <strong>fight</strong> map<br><small>Follow the conquest of your team</small>",
+"GYMS_NO_DEFENDER": "No Defender yet",
 "HOME": "Home",
 "HOUR": "hour",
 "HOURS": "hours",
 "INCITY": "in %s",
 "INSTINCT": "Instinct",
+"IV_ATTACK": "IV Attack",
+"IV_DEFENSE": "IV Defense",
+"IV_STAMINA": "IV Stamina",
```

Comment:
Please use `ATTACK`, `DEFENSE` and `STAMINA` only and just prepend `"IV "`. We don't have to translate this twice.